app/Services/FileSystem/Swift/SwiftServiceProvider.php

```diff
@@ -43,6 +74,8 @@ public function boot()
         Storage::extend('swift', function ($app, $config) {
 
             try {
```

Comment:
let adds a return null if the config object has not set the auth url the region or the container

Reply:
done